### DISCUSSION TITLE
Readme not updating

### DISCUSSION THREAD

**Discussion by nero-sensei** [athul/waka-readme#94] [Q&A] [answered]
While running the action there is an error stating.

`ln. 416 CRITICAL 403 {"message": "Resource not accessible by integration", "documentation_url": "https://docs.github.com/rest/reference/repos#create-or-update-file-contents"}`

![image](https://user-images.githubusercontent.com/77715088/184538430-fb9219e1-c290-47dd-9813-f23ad1fe37e8.png)

any help would be appreciated.

**Comment by nandgator** [comment, marked as answer]
[HTTP 403](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403) is _forbidden_ request. Suggestions:

- Are you running this locally/ on a non-profile readme? If so,
    - is the GitHub token provided?
    - does the GitHub token have [proper scope](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token)?
- https://github.com/athul/waka-readme/pull/92 has been merged. Please try again.
